webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/style.scss

```diff
@@ -0,0 +1,9 @@
+ul#dropdown_send_post_options {
+    li[role="menuitem"][aria-disabled="true"] {
```

Comment:
Could these styles be applied directly to the menu component instead of this being a special case? I don't know how if it currently supports disabled elements, but I don't see why it couldn't

Reply:
By default it greys out, and doesn't bold disabled elements, both things which we wanna customise as per designs. Styling via CSS selectors is significantly easier, more readable and much more powerful than passing styles in JSX code to the component.

Reply:
Sorry, I meant to say "could this be applied to everywhere this Menu component is used instead of just this one?"

I can't think of anywhere else that we disable menu items, but I think it'd be good to ensure that that's consistently applied in the future. We could double check with UX on that, but I don't expect them to want this menu to be different from others

Reply:
We don't want to do it. You do want disabled menu items to appear disabled, right? This here in this component is the exception where I'm using a disabled menu item as a way to display a text with same font size, same height and everything. Setting this style globally will make all disabled and enabled elements look alike.

Reply:
Why do you want to use disabled styled here?

Reply:
Because I'm altering a disabled menu item. I want to display a text. I can use a div or span but will have to manage its all styling like width, height, font etc to match that of other menu items. So, I use a manu item and disabled it so it appears as a text and is disabled so you can't select it.

Reply:
Oh, I see. I didn't realize that that's why the menu item was disabled.

That seems reasonable to me then. We'll need a generic `Menu.Heading` component eventually for some other menus in the app, but I don't think we need to add that right now. The disabled menu item is a bit funny looking, but that's totally fine.

I also found that you can pass a `subheader` into the `MuiMenu`'s `MenuListProps`, but it also needs custom styling unless you use their `ListSubheader` component which defaults to the tiny text that Android menu headings have, so it has the same issue as using an arbitrary disabled div